tidy3d/plugins/microwave/auto_path_integrals.py

```diff
@@ -0,0 +1,76 @@
+"""Helpers for automatic setup of path integrals."""
+
+from ...components.geometry.base import Box
+from ...components.geometry.utils import SnapBehavior, SnapLocation, SnappingSpec, snap_box_to_grid
+from ...components.grid.grid import Grid
+from ...components.lumped_element import LinearLumpedElement
+from ...components.types import Direction
+from .path_integrals import (
+    CurrentIntegralAxisAligned,
+    VoltageIntegralAxisAligned,
+)
+
+
+def path_integrals_from_lumped_element(
```

Comment:
Is the “path integral” terminology standard here? I would call this a line integral

Reply:
If I could go back in time, I might have called these tools `LineIntegral` s, but I think those terms are interchangeable anyways. I like path because it indicates that it could be a line, curve, or closed loop.

Reply:
I think the only negative is that there might be a slight confusion with path integrals from quantum mechanics.